test/widgets/recent_dm_conversations_test.dart

```diff
@@ -106,6 +108,24 @@ void main() {
       await tester.pumpAndSettle();
       check(tester.any(oldestConversationFinder)).isTrue(); // onscreen
     });
+
+    testWidgets('opens new DM sheet on FAB tap', (tester) async {
+      await setupPage(tester, users: [], dmMessages: []);
+      final fab = find.ancestor(
```

Comment:
nit: Let's [avoid abbreviations](github.com/flutter/flutter/blob/master/docs/contributing/Style-guide-for-Flutter-repo.md#avoid-abbreviations) like this (here and in the test name) when possible, unless it comes from an outside source like the Figma design. In this case, we are just referring to the new DM button widget.